### ISSUE TITLE
[Success Image Report]: Windows 10 Version 1909

### ISSUE THREAD

**Issue by gottalovebrando** [ventoy/Ventoy#1287]
### Official Website List

- [X] I have checked the list in official website and the image file is not listed there.

### Ventoy Version

ventoy-1.0.61-windows

### BIOS Mode

Both

### Partition Style

GPT

### Image file name

Windows 10 Version 1909 Nov 2019 x32 and x64 Home, Home N, Home Single Language, Education, Education N, Pro, Pro N

### Image file checksum type

SHA256

### Image file checksum value

1b80b32fca10312d4a207d178fd2d6db6ce7e56c9199e2403ce59394b1df52d8

### Test envrionment

Multiple (see details)

### More Details?

This image file booted successfully in Ventoy with 3 different machines when the x64 option was selected. Not sure how windows creates their ISO versions but it looks like there are many different checksums.
If desired, MD5 for the same ISO is 7c5771c16e5171bb751e38c105c6af4c

NOTE: machine #3 (secure boot off, UEFI mode), failed to boot once with an error:
BAD SYSTEM CONFIG INFO
Next boot under same conditions was a success.

Machine #1: Microsoft surface go model 1824	Only tested with secure boot in UEFI mode
Processor	Intel(R) Pentium(R) CPU 4415Y @ 1.60GHz   1.61 GHz
Installed RAM	4.00 GB (3.89 GB usable)
System type	64-bit operating system, x64-based processor
Installed OS (just in case)	Windows 10 Pro
	
Machine #2: HP Pavilion Laptop 14-ce0xxx	Only tested with secure boot in UEFI mode
Processor	Intel® Core™ i5-8250U CPU @ 1.60GHz × 8 
Installed RAM	15.5 GiB (1x16GB)
System type	64-bit operating system, x64-based processor
Installed OS (just in case)	Ubuntu 20.04.3 LTS
	
Machine #3: Gigabyte Tech. H61MA-DV2 Motherboard (desktop)	secure boot disabled (incapable?), UEFI and legacy BIOS capable
Processor	i3-2100 @ 3.1 GHz
Installed RAM	4 GB (2x2GB)

**Comment by ventoy** [comment]
Thanks. I have updated the page.

**Comment by LifeCanvasDev** [comment]
Hi. Is there an "official" way to report success and failure of various Operating Systems and tools booted via Ventoy?

I would like to contribute the following info about various OSs and tools that I've used with Ventoy:

Ventoy version used: ventoy-1.0.91-windows
BIOS Mode: UEFI; Secure Boot disabled
Partition Style: GPT
All of the ISO images were on a NTFS partition on 16GB ADATA microSD card

All of these boot attempts (successful and failed) and installs were done on bare-metal (no VMs) on a Lenovo ThinkPad T440p (64-bit Intel Core i5-4300M, 2x8GB RAM)


The following all booted and installed successfully from above-mentioned Ventoy-based media, onto a Samsung 860 EVO 1TB V-NAND 2.5 inch SATA SSD in the T440p:

artix-lxqt-runit-20230501-x86_64.iso
SHA256: 62f35ad979558371b51e06f2c72eadfff9ee3ba810285dcb3a27b5792cf0f048
Artix did not boot with the "normal" mode, but did so (and installed) with "grub2" mode

Fedora-Everything-netinst-x86_64-38-1.6.iso
SHA256: 4d042dedc8886856db10bc882074b84dcce52f829ea7b3f31d8031db8d84df20
normal mode

FreeBSD-13.2-RELEASE-amd64-disc1.iso
SHA256: b76ab084e339ee05f59be81354c8cb7dfadf9518e0548f88017d2759a910f17c
normal mode

refind-cd-0.14.0.2.iso (rEFInd CD version) - installed to EFI
normal mode


The following also booted successfully and could be used without any problems in their live environments (on the same Lenovo T440p), but were not installed:

gparted-live-1.5.0-1-amd64.iso
SHA256: 3226629b202b8b19848a196f55c522ed6eaec8199830a9449002987a2276e5b1

PeppermintOS-Devuan_64.iso 2022-11-06
SHA512:  6d6847531255ed559f0aefd13e1a9ab6c5d380794cfeb00efd59cba2f96a8fb3ca7e4e838c91ea161955f05d5dc1b56f4eeaf8ff402d99d1e25aa5f74061c5c6

kali-linux-2023.1-live-amd64.iso
SHA256: b7c93ff333b6e3909a6e28de5408a916cbea2b5c2371e4b76ec12900d1158bc8

void-live-x86_64-musl-20221001-base.iso (using runit init system)
SHA256: bd3c17365cc94019bd8b1d72418feee7ba0004807f7d67292254c62cf348343c

HelenOS-0.12.1-amd64.iso


fails:

openbsd-amd64-install73.iso
SHA256: fdf1210ffe87213eeca5f1d317e8b19364cbae83545cdfc7845098a53fc79a60
normal and grub2 mode: "No bootfile found for UEFI!
Maybe the image does not support x64 UEFI"

slax-64bit-15.0.1.iso (Slackware 15-based Slax Linux)
No bootfile found for UEFI!
Maybe the image does not support x64 UEFI

S15Pup64-22.12-230513.iso (Slack 15 Puppy Linux 64bit; kernel 5.15.110)
SHA512: 16c3352300beffa75dd571b364fb0f1bc5f1d74ed3b6ec143701eac2aadbdf6206b98058d3a9e4d98d877417c56e586ba16c8680fb3db7ea5f517e8399615554
"Finding puppy main sfs file."
"Dumping last lines of /tmp/bootinit.log..."
"ventoy: not a valid block device"
"ventoy on /mnt/pdrv as    mount failed."
"6: ONE_PART=ventoy ONE_TRY_FN= PDRV="
"[   4.057755] usb 3-12: New USB device found, idVendor..."
"ventoy /puppy_s15pup64.12.sfs not found."
"Error is too critical, dropping out to console..."

then I tried again with grub2 mode and Puppy option 2: Copy SFS files to RAM
same failed result
